shared/constants/field/myinfo/index.ts

```diff
@@ -34,6 +37,7 @@ export const types: MyInfoFieldBlock[] = [
       'The registered name of the form-filler. This field is verified by ICA for Singaporeans/PRs & foreigners on Long-Term Visit Pass, and by MOM for Employment Pass holders.',
     fieldType: BasicField.ShortText,
     previewValue: 'PHUA CHU KANG',
+    previewIsDisabled: true,
```

Comment:
i wonder if we should discern what is or is not disabled on runtime (like how public forms are displayed) based on the `verified` prop instead of hardcoding it here.

Also wondering if we should actually disable the state in the builder, if it's disabled won't the admins be unable to preview myinfo options properly?

Reply:
just discussed with @mantariksh and @pearlyong , tldr will leave the MyInfo fields editable so will close this PR. Have closed the [related issue](https://github.com/opengovsg/FormSG/issues/4362)